roles/dropbear_luks_unlock/defaults/main.yml

```diff
@@ -3,8 +3,19 @@ dropbear_luks_required_packages:
   - "dropbear-initramfs"
   - "cryptsetup-initramfs"
 
-dropbear_initramfs_config_path: "/etc/dropbear-initramfs/config"
-dropbear_initramfs_authorized_keys_path: "/etc/dropbear-initramfs/authorized_keys"
+dropbear_crypttab_path: "/etc/crypttab"
+dropbear_initramfs_base_paths:
+  Debian:
+    10: "/etc/dropbear-initramfs"
+    11: "/etc/dropbear-initramfs"
+    12: "/etc/dropbear/initramfs"
+  default: "/etc/dropbear-initramfs"
+
+dropbear_initramfs_base_path: "{{ dropbear_initramfs_base_paths[ansible_distribution]
+                                                               [ansible_distribution_major_version | int]
```

Comment:
I have done the opposite, having String indices, as f.ex. 11.5 or a release name in other distros might not be an integer, while an integer can be represented in a string. Also, have not yet found a good way to have a "ansible_distribution" level default...

Reply:
Also, it is more common to have strings as indices in yaml dicts. both work though

Reply:
hmm yeah release names could resolve this, but that'll fall apart on rolling release OS and OS without proper names. For `Debian` this *should* work tho.

Reply:
True, but i would change the 10, 11, 12 to "10", "11", "12" and cast to string, would then also work for release code names without changing much of the logic. Arch's `distribution_major_version` is probably just not set..?

Reply:
Sounds reasonable. Will change asap